src/components/Header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+
+import './Header.css';
+import { HEADER as IconHeader } from './iconTypes';
```

Comment:
I would stick to some constants convention to avoid aliasing, if you import such, it worth being more specific when exporting it. (see my next comment in `iconTypes` file)